### DISCUSSION TITLE
Lookdata condition syntax change?

### DISCUSSION THREAD

**Discussion by kamurj** [AutomaApp/automa#1460] [Q&A]
Sorry, the title should have been Loopdata and not lookdata.

This used to work until recently, but now it no longer does.

![image](https://github.com/AutomaApp/automa/assets/8271715/4cc2d4fa-a5cb-4e77-9590-dd2d849d3295)

Did something change and I should not be using loopData@zloop_id.title syntax?

**Comment by Delusionnnn** [comment]
Try this syntax.

{{loopData.loopID.path}}

**Comment by kamurj** [comment]
Thank you. I will try it.
